lib/quby/questionnaires/entities/question.rb

```diff
@@ -166,9 +166,6 @@ def initialize(key, options = {})
             @validations << {type: :valid_float, explanation: options[:error_explanation]}
           elsif @type == :integer
             @validations << {type: :valid_integer, explanation: options[:error_explanation]}
-          elsif @type == :date
```

Comment:
Ik zie nu dat je dat in de initialisatie van de question zelf doet, maar is het niet beter om dit soort standaard validaties op 1 plek in de code te houden?